pkg/operator/ceph/cluster/controller.go

```diff
@@ -780,6 +782,53 @@ func (c *ClusterController) onDeviceCMUpdate(oldObj, newObj interface{}) {
 // Delete event functions
 // ************************************************************************************************
 
+func (c *ClusterController) dataDirHostPathCleanUp(clust *cephv1.CephCluster) {
+	logger.Infof("starting data dir host path clean up for cluster - %+v", clust)
```

Comment:
Just print the name not the full object.

Reply:
Use `dataDirHostPath` as one word since it is a named reference to the CephCluster spec

```suggestion
	logger.Infof("starting dataDirHostPath clean up for cluster %q, clust.Namespace)
```

Reply:
Updated the log message.